plugins/woocommerce-admin/client/customer-effort-score-tracks/data/reducer.js

```diff
@@ -14,6 +16,25 @@ const reducer = ( state = DEFAULT_STATE, action ) => {
 				...state,
 				queue: action.queue,
 			};
+		case TYPES.HIDE_CES_MODAL:
+			return {
+				...state,
+				showCESModal: false,
+				cesModalData: undefined,
+			};
+		case TYPES.SHOW_CES_MODAL:
+			const cesModalData = {
```

Comment:
This as well as the other data store calls will need to be updated as outlined in other comments.